webview/src/plots/components/ZoomedInPlot.tsx

```diff
@@ -40,6 +43,12 @@ export const ZoomedInPlot: React.FC<ZoomedInPlotProps> = ({
   isTemplatePlot,
   openActionsMenu
 }: ZoomedInPlotProps) => {
+  const isCustomPlot = !isTemplatePlot
+  const hasSmoothing =
+    isTemplatePlot &&
+    (props.spec as { params?: { name: string }[] }).params?.[0]?.name ===
+      'smooth'
```

Comment:
[C] This is pretty brittle but will do for now. Will discuss at the next cross-team whether or not we want to push this back in to `dvc-render`.

Reply:
worst case scenario - if the spec changes then `makePlotZoomOnWheel` will fall through to the default. The zoom/pan behaviour won't work but the template will still render.